phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -71,7 +74,7 @@ public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
-    public void showDetailsView(Node nodeToShow) {
+    public void showDetailsView(DetailsView nodeToShow) {
```

Comment:
I don't think we should change this (or do you use any special functionality of `DetailsView` here?)